release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3504,3 +3504,27 @@ $ oc adm release info 4.10.36 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-37"]
+=== RHBA-2022:6901 - {product-title} 4.10.37 bug fix update
+
+Issued: 2022-10-18
+
+{product-title} release 4.10.37, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:6901[RHBA-2022:6901] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:6899[RHBA-2022:6899] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.37 --pullspecs
+----
+
+[id="ocp-4-10-37-bug-fixes"]
+==== Bug fixes
+
+* Previously, adding an IP address to one or more control plane nodes caused the Etcd Operator to fail to regenerate etcd serving certificates for the node. With this update, the Etcd Operator regenerates serving certificates for changes to an existing node. (link:https://issues.redhat.com/browse/OCPBUGS-1758[*OCPBUGS-1758*])
```

Comment:
Should it be etcd cluster Operator? Just curious based on https://docs.openshift.com/container-platform/4.11/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html